frontend/src/MuiForms/TagSelectorBeta.tsx

```diff
@@ -0,0 +1,78 @@
+import { Autocomplete, Box, Chip, Stack, TextField, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+
+export default function TagSelector(props: any) {
+  const { onChange, value: currentValue, label, formContext } = props
+
+  const _onChange = (_event: React.SyntheticEvent<Element, Event>, newValues: any) => {
```

Comment:
Can `newValues` be given a better type? I imagine generics could come in handy here